puppet/modules/logfile/manifests/init.pp

```diff
@@ -0,0 +1,26 @@
+#
```

Comment:
i put it in site_config originally, then moved it because it didn't seem to fit (it is not a base config related item at all). how about we start 'leap' module that is like 'common' but leap-specific?